README.md

```diff
@@ -1,9 +1,8 @@
 # Minetest Game Internationalization Mod
-MTG 0.5.0-dev (2017-08-28), License: MIT (code), CC 0 (translations)
+MTG 5.0.1 (2017-08-28), License: MIT (code), CC 0 (translations)
 
 ## Description
-A Minetest mod adding translations to the default game of Minetest. You just have to install the
-[intllib](https://github.com/minetest-mods/intllib/) mod and enable both in your world and all item
+A Minetest mod adding translations to the default game of Minetest. You just have enable both in your world and all item
```

Comment:
> You just have enable both in your world and all item

This makes no sense.

Reply:
Proposed another sentence.

Reply:
Add a dot at the end of the line.
I think it'll look better.